examples/inference-deployments/README.md

```diff
@@ -1,11 +1,36 @@
 # MosaicML Inference Model Handlers
 
-This repo contains model handler implementations for various models. They can be used out-of-the box with the yamls provided using the MosaicML inference service.
+The following instructions will guide you on how to get started with [MosaicML’s inference service](https://www.mosaicml.com/blog/inference-launch). 
+
+In this folder, we provide examples of model handler implementations for various models. They can be used out-of-the box with the yamls provided using the MosaicML inference service. 
 
 ## Getting Started
 
-First, you'll need access to MosaicML's inference service. You can request access [here](https://forms.mosaicml.com/demo).
+Before using the inference service, you must request access [here](https://forms.mosaicml.com/demo).
+
+Once you have access, 
+
+1. Follow instructions [here](https://docs.mosaicml.com/projects/mcli/en/latest/quick_start/getting_started.html) to install mosaicml-cli, our command line interface that will allow you to deploy models and run inference on them.
+2. Once you have `mcli` set up, the [Inference Docs](https://docs.mosaicml.com/projects/mcli/en/latest/quick_start/quick_start_inference.html) will give you a high level overview of how you can deploy your model and interact with it.
+3. Now, you are ready to look at the README for each of the models in this repo to start running inference on them!
+
+
+## Content
+We have provided examples for 3 different model types in this repo:
+
+1. MPT series
+2. Instructor 
+3. Mosaic diffusion
+
+Each of these model directories have model handlers and yaml files.
+
+### Model Handlers
+
+Model handlers define how your model is loaded and how the model should be run when receiving a request. The model handlers are expected to be a class that implements a `predict` function and optionally a `predict_stream` function if you'd like your deployment to support streaming outputs. For more details about the structure of the model handlers, please refer to the [mcli docs](https://docs.mosaicml.com/projects/mcli/en/latest/inference/deployment_features.html#model-handlers)
+
+If a model that you'd like to deploy isn't supported by one of the existing model handlers, you can flexibly configure the behavior of your models in the server by implementing your own model handler. 
+
 
-After getting started, follow the [MCLI Docs](https://docs.mosaicml.com/projects/mcli/en/latest/quick_start/quick_start_inference.html) for quick start instructions.
+### YAMLs
```

Comment:
Can we add a brief section here mentioning the inference image, high level what it contains, and noting that it is pinned in all the examples?